modules/apps/object/object-rest-test/src/testFunctional/macros/CustomObjectAPI.macro

```diff
@@ -89,7 +119,12 @@ definition {
 		}
 
 		if (isSet(keywords)) {
-			var body = StringUtil.add("${body},", "\"keywords\": [\"${keywords}\"]", "");
+			if (${keywords} == "") {
```

Comment:
Just share that if we use ${keywords} and pass keyword="[]" then there is no need to use if(${keyword}=="") if/else condition anymore, but I'm ok with the current codes :smile:

Reply:
？

Reply:
Already shared in slack

Reply:
I have tried keyword="["k1"]", it can not work